src/xdocs/config_naming.xml

```diff
@@ -1226,19 +1226,11 @@ public boolean equals(Object o) {
         </source>
         <p>
           The following configuration checks that the parameters always start with two
-          lowercase characters and, in addition, that public method parameters cannot be one character
```

Comment:
it seems to me we are removing the original purpose of this example, which is how to configure this check differently between public and non-public naming conventions.
I think we should restore it. It should be similar to google config, one should be `public` alone and the other should have all the other modifiers.